examples/layer-browser/src/app.js

```diff
@@ -197,38 +207,46 @@ class App extends PureComponent {
   /* eslint-enable max-depth */
 
   _getModelMatrix(index, coordinateSystem) {
-    // the rotation controls works only for layers in
-    // meter offset projection mode. They are commented out
-    // here since layer browser currently only have one layer
-    // in this mode.
     const {settings: {separation}} = this.state;
     const modelMatrix = new Matrix4().translate([0, 0, 1000 * index * separation]);
 
-    // if (coordinateSystem === COORDINATE_SYSTEM.METER_OFFSETS) {
-    //   const {settings: {rotationZ, rotationX}} = this.state;
-    //   modelMatrix.rotateZ(index * rotationZ * Math.PI);
-    //   modelMatrix.rotateX(index * rotationX * Math.PI);
-    // }
+    switch (coordinateSystem) {
+    case COORDINATE_SYSTEM.METER_OFFSETS:
+    case COORDINATE_SYSTEM.IDENTITY:
+      const {settings: {rotationZ, rotationX}} = this.state;
+      modelMatrix.rotateZ(index * rotationZ * Math.PI);
+      modelMatrix.rotateX(index * rotationX * Math.PI);
+      break;
+    default:
+      // Rotations don't work well for layers in lng lat coordinates
+      // since the origin is far away.
+      // We could rotate around current view point...
+    }
 
     return modelMatrix;
   }
 
-  _renderNoTokenWarning() {
-    /* eslint-disable max-len */
-    return (
-      <div id="no-token-warning">
-        <h3>No Mapbox access token found.</h3>
-        Read <a href="http://uber.github.io/deck.gl/#/documentation/overview/getting-started#note-on-map-tokens-">"Note on Map Tokens"</a> for information on setting up your basemap.
-      </div>
-    );
-    /* eslint-disable max-len */
-  }
-
   _getViewports() {
-    const {width, height, mapViewState, settings: {multiview}} = this.state;
+    const {
+      width, height, mapViewState, orbitViewState,
+      settings: {infovis, multiview}
+    } = this.state;
+
+    if (infovis) {
```

Comment:
Use infovis to switch between geospatial vs non-geospatial examples everywhere, not very scalable for new non-geospatial example in the future.

Reply:
> 

For sure this was just intended to get us started.

FWIW, there is a `layerFilter` function which automatically shows all `COORDINATE_SYSTEM.IDENTITY` when in infovis mode. No need to add more if statements to each example, so should be somewhat "scalable" in that sense.

Although there are probably an issue if the demo layers use data sets with very different extents, we may want to scale them to similar size (maybe using modelMatrix).

I also thought about putting two viewports on top of each other so that all layers are visible. We can refine it like that later on.